cgotorch/cgotorch.h

```diff
@@ -27,6 +28,32 @@ void Step(Optimizer opt);
 void AddParameters(Optimizer opt, Tensor *tensors, int length);
 void Optimizer_Close(Optimizer opt);
 
+// transform APIs
+typedef void *Transform;
+Transform Normalize(double mean, double stddev);
+Transform Stack();
+
+// dataset APIs
+typedef void *Dataset;
+Dataset MNIST(const char *data_root);
+
+// Add transform on dataset
+void AddNormalize(Dataset dataset, Transform transform);
```

Comment:
Is this function calling a method of C++ type Dataset? If so, should we rename it into `Dataset_AddNormalize`?

Reply:
Good catch, I  will do that in the next PR.